FORECAST-WEATHER-API-BLUEPRINT.apib

```diff
@@ -75,6 +75,20 @@ For multiple locations, its recommended to just use the point compression.
 The forecasts in the response collection are "flat" to allow easy streaming and filtering on client side.
 The forecasts in the response are ordered ascending by field 'validFrom'.
 
+It is important to use encoded timestamps when doing the requests to Point Forecast Service by for example POSTMAN or browser.
+To encode properly the timestamp
```

Comment:
To encode properly the timestamp:
1. Go to https://www.urlencoder.org/ site.
2. Paste desired timestamp.
3. Without changing any parameters click "> ENCODE <" button.
4. Use generated encoded timestamp in the url.